.travis.yml

```diff
@@ -7,7 +7,7 @@ android:
   components:
     - tools
     - platform-tools
-    - build-tools-27.0.0
+    - build-tools-27.0.3
     - android-19
```

Comment:
I noticed that we also have `android-16` here in https://github.com/bugsnag/bugsnag-android, should that be added here as well?

Reply:
I'll create a ticket to address that separately - there may have been some reason we skipped setting up 16 originally.